plugins/imfile/imfile.c

```diff
@@ -490,8 +490,12 @@ static rsRetVal enqLine(lstn_t *const __restrict__ pLstn,
 	MsgSetTAG(pMsg, pLstn->pszTag, pLstn->lenTag);
 	msgSetPRI(pMsg, pLstn->iFacility | pLstn->iSeverity);
 	MsgSetRuleset(pMsg, pLstn->pRuleset);
-	if(pLstn->addMetadata)
+	if(pLstn->addMetadata) {
+		uchar sOffset[80];
```

Comment:
80 seems like overkill - 23 should suffice, or just use 32 for a nice power of 2.